bio/gatk3/indelrealigner/wrapper.py

```diff
@@ -8,34 +8,39 @@
 from snakemake.shell import shell
 from snakemake_wrapper_utils.java import get_java_opts
 
+input_known = snakemake.input.known
```

Comment:
I would do this via a get, so that you don't get a key error if `known=""` is not specified (which is a valid choice, I think?):
```suggestion
input_known = snakemake.input.get("known", "")
```

Reply:
Never mind, I double checked the [old docs](https://github.com/broadinstitute/gatk-docs/blob/master/gatk3-tooldocs/3.8-0/org_broadinstitute_gatk_tools_walkers_indels_IndelRealigner.html) and this is indeed an obligatory parameter and thus a key error is fine! Sorry for the noise...

Reply:
No problem... :smile: